package.json

```diff
@@ -28,12 +28,12 @@
     "test": "node test/scss_to_json_tests.js"
   },
   "devDependencies": {
-    "grunt": "~0.4.5",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-clean": "^0.5.0"
+    "grunt": "^1.0.1",
+    "grunt-cli": "^1.2.0",
+    "grunt-contrib-clean": "^1.0.0"
   },
   "peerDependencies": {
-    "grunt": "~0.4.5"
```

Comment:
per https://github.com/ryanbahniuk/grunt-scss-to-json/pull/1 this probably wants to be 

```
"grunt": ">=0.4.0"
```